recipes-devtools/rust/rust-version.inc

```diff
@@ -1,6 +1,6 @@
 # Note: if you adjust this, you'll also need to change the hashes in
 # rust-source.bb
 SOURCE_NAME = "rust"
-PV = "1.10.0"
+PV = "1.12.0"
```

Comment:
Sure. This branch is literally me taking @jmesmon's work in #89 and pulling it into my own repo and rebasing it on top of master + the CI work I've been doing.